internal/parse/parser.go

```diff
@@ -177,15 +177,17 @@ func (p *Parser) skipByteFind(b byte) bool {
 	return false
 }
 
-// skipSpaces advances the parser jumping over consecutive spaces. It stops when
-// finding a non-space character. Returns true if the parser position was
-// actually changed, false otherwise.
-func (p *Parser) skipSpaces() bool {
+// skipBlanks advances the parser past spaces, tabs and newlines. Returns
+// whether the parser position was changed.
+func (p *Parser) skipBlanks() bool {
 	mark := p.pos
-	for p.pos < len(p.input) {
-		if p.input[p.pos] != ' ' {
-			break
-		}
+	blanks := map[byte]bool{
+		' ':  true,
+		'\t': true,
+		'\r': true,
+		'\n': true,
+	}
+	for p.pos < len(p.input) && blanks[p.input[p.pos]] {
```

Comment:
This is a longer and much more expensive version of this:
```
for ... {
    switch p.input[p.pos] {
    case ' ', '\t', ...:
        ...
    }
}
```

Reply:
Done. Thanks.